PRChecker/Shared/Models/OldPullRequest.swift

```diff
@@ -24,7 +24,16 @@ class OldPullRequest: AbstractPullRequest {
     }
     
     override var isRead: Bool {
-        false   // TODO: Can we get this data somewhere else?
+        // There is no `isRead` pre-Github Enterprise 3.0, so we need to fake it
+        guard let nodes = pullRequest.reviews?.nodes else {
+            return false     // No reviews at all
+        }
+        
+        guard let _ = nodes.last(where: { $0?.author?.login.lowercased() == viewingUser.lowercased() }) else {
+            return false     // Viewer has not reviewed
+        }
+        
+        return true
```

Comment:
I think `isRead` means "Does the current user read PR already?", and if that is indeed the case, then it probably is not related to `ReviewStatus`. Can we just return a `false` value for pre-Github version?

Reply:
Ok, I'll revert that change then.__